Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -92,6 +93,24 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
             activeContainer = container
         }
     }
+    
+    public func updateChromelessMode(with option: Options) {
+        if let chromelessMode = options[kChromeless] as? Bool {
```

Comment:
There's propably a typo here, and you used the stored property `options` instead of the parameter `option`, which I guess was the original intention. 

But complementing my comment on the previous line, I think we should be using the stored property 😉